### ISSUE TITLE
Type enum.Enum is not serializable

### ISSUE THREAD

**Issue by aabrodskiy** [andrewthetechie/pydantic-aioredis#153]
One of my models has Enum fields in it, which can't seem to be currently handled and throw a type error.

Here is my simplified example:
```
class SomeType(enum.Enum):
    Interval = 'interval'

class SomeModel(Model):
    _primary_key_field: str = 'id'    
    id: str
    type: SomeType
```

Error message:
File "/home/.../.local/lib/python3.9/site-packages/pydantic_aioredis/abstract.py", line 80, in json_default
    raise TypeError("Type %s not serializable" % type(obj))
TypeError: Type <enum 'SomeType'> not serializable

**Comment by andrewthetechie** [comment]
Hi @aabrodskiy, thanks for the report.

One way to handle this would be to override serialize_partially to add your own serializer https://github.com/andrewthetechie/pydantic-aioredis/blob/main/pydantic_aioredis/abstract.py#L83-L107

I'll take a look at just adding enum support, but would also be happy for a PR!

**Comment by andrewthetechie** [comment]
@aabrodskiy I was able to add support for enums. 

Its included in [v0.5.0](https://github.com/andrewthetechie/pydantic-aioredis/releases/tag/v0.5.0)
